lib/workers/branch/get-updated.ts

```diff
@@ -264,16 +264,27 @@ export async function getUpdatedPackageFiles(
         config,
       });
       if (is.nonEmptyArray(results)) {
-        updatedPackageFiles.push(packageFile);
+        let updated = false;
         for (const res of results) {
           const { file, artifactError } = res;
           // istanbul ignore else
-          if (file) {
+          if (
+            file?.type === 'addition' &&
+            file.contents !== '' && // only check if artifact is non-empty --> ignore git-submodule artifacts
+            (await getFile(
```

Comment:
I still think we get into trouble when the file is the gradle wrapper jar. Which is binary.